spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowTaskExecutionDao.java

```diff
@@ -0,0 +1,131 @@
+import org.springframework.util.Assert;
+import org.springframework.util.StringUtils;
+
+/**
+ * Stores Task Execution Information to a JDBC DataSource. Mirrors the {@link JdbcTaskExecutionDao}
+ * but contains Spring Cloud Data Flow specific operations. This functionality might
+ * be migrated to Spring Cloud Task itself.
+ *
+ * @author Gunnar Hillert
+ */
+public class JdbcDataflowTaskExecutionDao implements DataflowTaskExecutionDao {
+
+	private static final Logger logger = LoggerFactory.getLogger(JdbcDataflowTaskExecutionDao.class);
+
+	/**
+	 * SELECT clause for task execution.
+	 */
+	public static final String SELECT_CLAUSE = "TASK_EXECUTION_ID, "
+			+ "START_TIME, END_TIME, TASK_NAME, EXIT_CODE, "
+			+ "EXIT_MESSAGE, ERROR_MESSAGE, LAST_UPDATED, "
+			+ "EXTERNAL_EXECUTION_ID, PARENT_EXECUTION_ID ";
+
+	/**
+	 * FROM clause for task execution.
+	 */
+	public static final String FROM_CLAUSE = "%PREFIX%EXECUTION";
+
+	/**
+	 * WHERE clause for running task.
+	 */
+	public static final String RUNNING_TASK_WHERE_CLAUSE = "where TASK_NAME = :taskName AND END_TIME IS NULL ";
+
+	/**
+	 * WHERE clause for task name.
+	 */
+	public static final String TASK_NAME_WHERE_CLAUSE = "where TASK_NAME = :taskName ";
+
+	private final NamedParameterJdbcTemplate jdbcTemplate;
+
+	private String tablePrefix = TaskProperties.DEFAULT_TABLE_PREFIX;
+
+	/**
+	 * Initializes the JdbcTaskExecutionDao.
+	 * @param dataSource used by the dao to execute queries and update the tables.
+	 * @param tablePrefix the table prefix to use for this dao.
+	 */
+	public JdbcDataflowTaskExecutionDao(DataSource dataSource, String tablePrefix) {
+		this(dataSource);
+		Assert.hasText(tablePrefix, "tablePrefix must not be null nor empty");
+		this.tablePrefix = tablePrefix;
+	}
+
+	/**
+	 * Initializes the JdbTaskExecutionDao and defaults the table prefix to
+	 * {@link TaskProperties#DEFAULT_TABLE_PREFIX}.
+	 * @param dataSource used by the dao to execute queries and update the tables.
+	 */
+	public JdbcDataflowTaskExecutionDao(DataSource dataSource) {
+		Assert.notNull(dataSource, "The dataSource must not be null.");
+		this.jdbcTemplate = new NamedParameterJdbcTemplate(dataSource);
+	}
+
+	private static final String DELETE_TASK_EXECUTIONS = "DELETE FROM %PREFIX%EXECUTION "
+			+ "WHERE task_execution_id in (:taskExecutionIds)";
+
+	private static final String DELETE_TASK_EXECUTION_PARAMS = "DELETE FROM %PREFIX%EXECUTION_PARAMS "
+			+ "WHERE task_execution_id in (:taskExecutionIds)";
+
+	private static final String DELETE_TASK_TASK_BATCH = "DELETE FROM %PREFIX%TASK_BATCH "
+			+ "WHERE task_execution_id in (:taskExecutionIds)";
+
+	@Override
+	public void deleteTaskExecutionsByTaskExecitionIds(Set<Long> taskExecitionIds) {
+		final MapSqlParameterSource queryParameters = new MapSqlParameterSource()
+				.addValue("taskExecutionIds", taskExecitionIds);
+		final String query = getQuery(DELETE_TASK_EXECUTIONS);
+		logger.info(">>>>>>>." + query);
```

Comment:
This logger info needs to be removed or updated.    And if kept probably needs to be debug.

Reply:
Removed